src/sagemaker/model.py

```diff
@@ -235,6 +235,8 @@ def deploy(self, initial_instance_count, instance_type, accelerator_type=None, e
                 If True, this will deploy a new EndpointConfig to an already existing endpoint and delete resources
                 corresponding to the previous EndpointConfig. If False, a new endpoint will be created. Default: False
             tags(List[dict[str, str]]): The list of tags to attach to this specific endpoint.
+            kms_key (str): The KMS key that is used to encrypt the data on the storage volume attached
```

Comment:
Let's make it clear that this is the arn of the kms_key

Reply:
done